cmd/internal/printconfig/command.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package printconfig
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func printConfigurations(cmd *cobra.Command, v *viper.Viper, includeEmpty bool) {
+	keys := v.AllKeys()
+	sort.Strings(keys)
+
+	for _, key := range keys {
+		value := v.Get(key)
+		source := "default"
+		if v.IsSet(key) {
+			source = "user-assigned"
+		}
+
+		if includeEmpty {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s: %v=%s\n", source, key, value)
+		} else if value != nil && value != "" {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s: %v=%s\n", source, key, value)
```

Comment:
why is value printed with special characters? `%!s(bool=false)`

Reply:
but not always:
```
default: processor.zipkin-compact.workers=%!s(int=10)
default: reporter.grpc.retry.max=3
```

Reply:
It was printing in this format because I was using the "v.Get()" and now I change to a v.GetString() and corrected this format